doc/version_history.rst

```diff
@@ -1,11 +1,30 @@
-.. py:currentmodule:: lsst.ts.MTDomeTrajectory
+.. py:currentmodule:: lsst.ts.mtdometrajectory
 
-.. _lsst.ts.MTDomeTrajectory.version_history:
+.. _lsst.ts.mtdometrajectory.version_history:
 
 ###############
 Version History
 ###############
 
+v0.8.0
+------
+
+Changes:
+
+* `MockDome` update for ts_xml 10.1, which is required.
+* Rename package to ``ts_mtdometrajectory`` and Python namespace to ``lsst.ts.mtdometrajectory``.
+* Update to use ts_utils.
+* Modernize unit tests to use bare asserts.
+* Test black formatting with pytest-black instead of a custom unit test.
+
+Requires:
+
+* ts_salobj 6.3
+* ts_config_mttcs
+* ts_simactuators 2
+* ts_utils 1
+* IDL files for MTDomeTrajectory, MTDome, and MTMount built from ts_xml 10.0
```

Comment:
Doesn't it require `ts_xml` 10.1? I know it hasn't been released yet but your Jira ticket has a link to another ticket for releasing your packages as soon as `ts_xml` 10.1 gets released.